data/json/npcs/classes.json

```diff
@@ -131,6 +131,32 @@
       { "skill": "rifle", "bonus": { "rng": [ 0, 2 ] } }
     ]
   },
+  {
+    "type": "npc_class",
+    "id": "NC_MARLOSS_VOICE",
+    "name": "Marloss Voice",
+    "job_description": "I spread the Hymns so that peace and unity return to our world.",
+    "//": "THRESH_MYCUS is here only for faction AI purposes. This NPCS is not meant to be part of the fungaloid mind, they have just eaten marloss berries and maybe seeds or gelatin ",
+    "traits": [ 
+      { "group": "BG_survival_story_RURAL" },
+      { "group": "NPC_starting_traits" },
+      { "group": "Appearance_demographics" },
+      [ "THRESH_MYCUS", 100 ],
```

Comment:
This will cause the NPC to appear as a mycus in tilesets. 
For example, in RetroDays, they will look like this
![mycus](https://user-images.githubusercontent.com/42699974/57495430-983c8800-72bd-11e9-80ba-df246dcdd694.png)

Given that `his NPCS is not meant to be part of the fungaloid mind`, perhaps you could add another hidden trait that has the same effect.

Reply:
Thanks! Didnt even think to check tilesets.